src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeExpressionParsing.kt

```diff
@@ -65,17 +66,52 @@ class SnakemakeExpressionParsing(context: SnakemakeParserContext) : ExpressionPa
         var indents = if (argsOnNextLine) 1 else 0
         var argNumber = 0
         while (!myBuilder.eof()) {
-            if (matchToken(PyTokenTypes.STATEMENT_BREAK)) {
+            if (atToken(PyTokenTypes.STATEMENT_BREAK)) {
+                println(myBuilder.rawLookup(2))
+                nextToken()
+                println(myBuilder.rawLookup(1))
+                /*
+                 * IMPORTANT: PLEASE READ BEFORE MODIFYING OR DEBUGGING!
+                 * if you check token type here or anywhere else via any method that uses getTokenType(),
+                 * e.g. atToken(), matchToken(), etc.,
+                 * it might apply filter and change the token type,
+                 * while nextToken() does not do that,
+                 * it just increases the current lexeme number, doesn't look at the token
+                 */
+                if (indents == 0 &&
+                        !findTokenRawLookup(separatorTokenType) &&
+                        !findTokenRawLookup(PyTokenTypes.INDENT) &&
+                        !findTokenRawLookup(PyTokenTypes.INCONSISTENT_DEDENT) &&
+                        !findTokenRawLookup(PyTokenTypes.DEDENT)) {
```

Comment:
Seems here we do 4 almost same lookups (skipping all whitespaces each time) instead of 1 lookup with checking that element not from token set. 

Could we implement it like `Parsing.atAnyOfTokens()` or use token set here for 4 elements ?
I'd rather expect smth like
*  `findTokenRawLookup(separatorTokenType, PyTokenTypes.INDENT, PyTokenTypes.INCONSISTENT_DEDENT, PyTokenTypes.DEDENT)` 
or
* `findTokenRawLookup(some_reasonable_named_token_set)`